### PR TITLE
chore(protocol): change Hekla sharingPctg to 80% & gasIssuancePerSecond to 1000000

### PR DESCRIPTION
**Pull request by dantaik**
@davidtaikocha when we release 1.10.0, we can test that the fee sharing pctg will be 80%. Do you think taiko-geth already support this?

Maybe we should also change _gasIssuancePerSecond_ to a much smaller value for Hekla?

### CODE REVIEW

**Comment by davidtaikocha** [comment]
Its supported.

**Review by davidtaikocha — APPROVED** [review]

**Review by YoGhurt111 — APPROVED** [review]

File: packages/protocol/contracts/layer1/hekla/HeklaTaikoL1.sol [hunk 1]
```diff
@@ -22,7 +22,7 @@ contract HeklaTaikoL1 is TaikoL1 {
             maxAnchorHeightOffset: 64,
             baseFeeConfig: LibSharedData.BaseFeeConfig({
                 adjustmentQuotient: 8,
-                sharingPctg: 75,
+                sharingPctg: 80,
                 gasIssuancePerSecond: 5_000_000,
```
Comment (dantaik) [inline packages/protocol/contracts/layer1/hekla/HeklaTaikoL1.sol:26]:
```suggestion
                gasIssuancePerSecond: 1_000_000,
```
??

Comment (davidtaikocha) [reply packages/protocol/contracts/layer1/hekla/HeklaTaikoL1.sol:26]: Also supported

Comment (dantaik) [inline packages/protocol/contracts/layer1/hekla/HeklaTaikoL1.sol:26]:
```suggestion
                gasIssuancePerSecond: 1_000_000,
```

**Review by YoGhurt111 — APPROVED** [review]

**Pull request merged**